metagraph-plugin/README.md

```diff
@@ -36,25 +36,25 @@ pytest tests
 Examples
 ===========================
 
-Loading Katana Graph
+Loading An Example Graph
 ------------------
 
 ```
 import metagraph as mg
-import katana.local
-from katana.example_data import get_input
-katana.local.initialize()
-pg = katana.local.Graph(get_input('propertygraphs/rmat15_cleaned_symmetric'))
-katana_graph = mg.wrappers.Graph.KatanaGraph(pg)
+import pandas as pd
+import pytest
```

Comment:
I don't think pytest is used here and it doesn't make sense to import into an example.